src/App.css

```diff
@@ -1,2 +1,5 @@
 @import-normalize; /* bring in normalize.css styles */
 
```

Comment:
The GlobalStyle component just got merged in the last PR so we don't need app.css file. you can put font-family and other global settings inside global style file after normalize